app/components/TrackDetailsComponent/tests/index.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+// import { fireEvent } from '@testing-library/dom'
+import { renderWithIntl } from '@utils/testUtils';
+import TrackDetailsComponent from '../index';
+import { translate } from '@app/components/IntlGlobalProvider/index';
+
+describe('<TrackDetailsComponent />', () => {
+  it('should render and match the snapshot', () => {
+    const { baseElement } = renderWithIntl(<TrackDetailsComponent />);
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should contain 1 TrackDetailsComponent component', () => {
```

Comment:
Looks like theres a lot more at play in the TrackDetailsComponents with the carousel component. Can we add in tests to ensure that's getting rendered too?